sdk/identity/azure-identity/azure/identity/_internal/__init__.py

```diff
@@ -52,12 +52,16 @@ def _scopes_to_resource(*scopes):
 
 
 __all__ = [
+    "_scopes_to_resource",
```

Comment:
It looks weird we want to expose a method start with '_'

Reply:
This module is `azure.identity._internal`, not part of the public API, so this is only being exposed for internal use. Underscoring it allows a public module to import it without accidentally implying it might be part of that module's public surface.